extension/src/test/suite/setup/util.ts

```diff
@@ -1,124 +1,60 @@
 import { join } from 'path'
 import { EventEmitter, commands } from 'vscode'
 import { Disposer } from '@hediet/std/disposable'
-import { fake, SinonSpy, stub } from 'sinon'
+import { fake, stub } from 'sinon'
 import { ensureDirSync } from 'fs-extra'
 import * as FileSystem from '../../../fileSystem'
 import { Setup } from '../../../setup'
 import * as Runner from '../../../setup/runner'
 import * as WorkspaceFolders from '../../../vscode/workspaceFolders'
-import { buildDependencies, mockDisposable } from '../util'
+import { buildDependencies } from '../util'
 import * as AutoInstall from '../../../setup/autoInstall'
-import { DvcReader } from '../../../cli/dvc/reader'
-import { DvcExecutor } from '../../../cli/dvc/executor'
-import { GitReader } from '../../../cli/git/reader'
-import { GitExecutor } from '../../../cli/git/executor'
 import { InternalCommands } from '../../../commands/internal'
 import { WorkspaceExperiments } from '../../../experiments/workspace'
-import { DvcRunner } from '../../../cli/dvc/runner'
 import { StopWatch } from '../../../util/time'
 import { WorkspaceScale } from '../../../telemetry/collect'
 import { dvcDemoPath } from '../../util'
 import { Config } from '../../../config'
-import { Resource, ResourceLocator } from '../../../resourceLocator'
+import { Resource } from '../../../resourceLocator'
 import { MIN_CLI_VERSION } from '../../../cli/dvc/contract'
+import { Status } from '../../../status'
 
 export const TEMP_DIR = join(dvcDemoPath, 'temp-empty-watcher-dir')
 
-const buildSetupDependencies = (
-  disposer: Disposer,
-  mockDvcRoot: string | undefined,
-  mockGitRoot: string | undefined,
-  noGitCommits = true
-) => {
```

Comment:
[F] Now that we are using `InternalCommands` we have to build the dependencies in the normal way.